index.html

```diff
@@ -601,48 +601,48 @@ <h3>Processing Model</h3>
   <p>
     This section outlines the steps the user agent must take when implementing the Compute Pressure Observer API.
   </p>
-  <section id="create-record">
-    <h3>Create and populate a ComputePressureRecord</h3>
+  <section>
+    <h3>Queue a ComputePressureRecord</h3>
+    <p>
+      To <dfn>queue telemetry data</dfn> of [=target type=] |target|,
+      run these steps:
+    </p>
     <ol class="algorithm">
       <li>
-        Let |readings| be the latest reading of the [=platform collector=].
+        Let |record:ComputePressureRecord| be the result of running [[[#create-record]]] for |target|.
       </li>
       <li>
-        Let |record:ComputePressureRecord| be a new {{ComputePressureRecord}} instance.
+        [=Append=] it to |observer|.{{ComputePressureObserver/[[QueuedRecords]]}}.
       </li>
-      <aside class="note">
-        TODO: https://github.com/WICG/compute-pressure/issues/46
-      </aside>
       <li>
-        Let |record|.|state:PressureState| be an [=implementation-defined=] state given
-        the data available from the [=platform collector=].
+        Let |document:Document| be the [=environment settings object/responsible document=] of the [=current settings object=].
       </li>
       <li>
-        Return |record|.
+        If |document| is not {{undefined}} and is not [=Document/fully active=], about these steps.
+      </li>
+      <li>
+        [=Queue a compute pressure observer task=].
       </li>
     </ol>
   </section>
-  <section>
-    <h3>Queue a ComputePressureRecord</h3>
-    <p>
-      To queue a {{ComputePressureRecord}} for a {{ComputePressureObserver}}
-      |observer:ComputePressureObserver|, given a [=document=] |document:Document|, run these steps:
-    </p>
+  <section id="create-record">
```

Comment:
It'd be good to have a `<dfn>` somewhere like in the section above so that we can also know what's referencing this definition.